pkg/auth/auth.go

```diff
@@ -41,27 +41,27 @@ func GetServicePrincipalToken(config *config.AzureConfig, aadEndpoint, resource
 	}
 
 	if config.UseManagedIdentityExtension {
-		klog.V(2).Info("using managed identity extension to retrieve access token")
```

Comment:
Do you think it would be beneficial to transfer Klog's log levels (v(2), v(3), v(4), etc) to their corresponding categories such as info, debug, trace, etc in mlog? If yes, what should be the appropriate mapping for it?